File: src/main/java/com/github/e2318501/anyban/BannedPlayer.java
package com.github.e2318501.anyban;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

import java.util.UUID;

@Getter
@RequiredArgsConstructor
public class BannedPlayer {
    private final UUID uuid;
    private final String name;
    private final String created;
    private final String source;
    private final String expires;
    private final String reason;
}

File: src/main/kotlin/com/github/tsuoihito/anyban/model/BannedPlayer.kt
package com.github.tsuoihito.anyban.model

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.*

data class BannedPlayer(
    @JsonProperty val uuid: UUID,
    @JsonProperty val name: String,
    @JsonProperty val created: String,
    @JsonProperty val source: String,
    @JsonProperty val expires: String,
    @JsonProperty val reason: String,
)
